### PR TITLE
Fix `.setVariable()` some elems too far out of view, add some debugging

### PR DESCRIPTION
**Pull request by plocket**
[`.focus()` seemed to work locally, but doesn't seem to work on GitHub for some reason. I'll try `.scrollIntoView(false)` next.]

1. I've been getting a bit tired of adding elaborate debugging stuff every time there's weird behavior. I added stuff that will be silent, but that I use all the time - coloring the DOM nodes. That should show up on screenshots if `DEBUG=1` in SECRETS, which no one knows about yet, so it shouldn't confuse anyone.
2. Puppeteer tries to scroll up to make an element visible before interacting with it. First, [that's known to not always be successful](https://github.com/puppeteer/puppeteer/issues/6462#issuecomment-717163409). Secondly, docassemble's header actually blocks the fields when they're at the top of the screen, so we need `false` in `.scrollIntoView(false)` to scroll things to the bottom of the screen instead. This concentrates on fixing it for the story table, though. It fixes it in `.setVariable()` and the functions called from it. I _think_ it It won't fix it for some other types of sentences.

Also added to the change log to prep for changes.

Just FYI, this passes, but actually shouldn't pass. The `showif_checkboxes_other` should be selecting `showif_checkboxes_nota_2`, but it's selecting `showif_checkboxes_nota_1`. No idea why yet.

Great to have this interview to test against!

### CODE REVIEW

File: lib/steps.js [hunk 1]
```diff
@@ -177,6 +177,7 @@ When(/^the user gets to ?(?:the)?(?: ?question)?(?: id)? "([^"]+)" with this dat
   // still have to be some automatic continuing.
   // It also seems harsh, but more reliable.
   // Ask developers what they'd like.
+  let count = 0;
```
Comment (niharikasingh) [inline lib/steps.js:180]: My comments on the other PR #168 carry over here too, since this one builds on that one.

File: lib/scope.js [hunk 2]
```diff
@@ -483,6 +483,14 @@ module.exports = {
       let { tag, type } = await handle.evaluate( elem => {
         return { tag: elem.tagName, type: elem.getAttribute('type') };
       });
+
+      if ( env_vars.DEBUG ) {
```
Comment (niharikasingh) [inline lib/scope.js:487]: This is a good idea. I like this!

File: lib/scope.js [hunk 3]
```diff
@@ -504,6 +512,9 @@ module.exports = {
     });
 
     if ( !disabled ) {
+      // da's header covers the top of the section containing the fields which means
+      // sometimes it blocks nodes from clicks. Scroll item to the bottom instead.
+      await handle.evaluate( elem => { elem.scrollIntoView(false); return elem.scrollTop });
```
Comment (niharikasingh) [inline lib/scope.js:517]: Do we need to `return elem.scrollTop` here? We don't seem to be using that value.

**Review by niharikasingh — APPROVED** [review]

File: lib/steps.js (hunk 1 above)
Comment (plocket) [reply lib/steps.js:180]: Made #171

File: lib/scope.js (hunk 3 above)
Comment (plocket) [reply lib/scope.js:517]: Added to #171, remove unused dev code

**Pull request merged**